examples/Loading and preprocessing DSGE data.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Instead of dropping the trends later, I think it would be better to make <code>df_trend = df.copy()</code> here, then save and plot them separately:</p><p> </p><pre class="ql-syntax" spellcheck="false">for col in log_cols:
&nbsp;&nbsp;&nbsp; cyc, trend = hpfilter(np.log(df[col].dropna()), lamb=1600)
&nbsp;&nbsp;&nbsp; df.loc[df[col].dropna().index, col] = cyc
&nbsp;&nbsp;&nbsp; df_trend.loc[df[col].dropna().index, col] = trend
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/gEconpy/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>